### DISCUSSION TITLE
Relationship column value

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#6345] [Help] [answered]
Can we set the value of  the relationship  column using the app ?

**Comment by melohagan** [comment]
Yep!
- https://docs.budibase.com/docs/form-steps
- https://docs.budibase.com/docs/referencing#example-creating-and-linking-two-table-entries-in-one-form

**Comment by rafikApp** [comment, marked as answer]
I have two related tables ( PIECES & INTERVENTIONS )
![Capture1](https://user-images.githubusercontent.com/106697208/174070397-fa3f2049-08c7-4639-9eec-c2598da54a6b.PNG)
ID_PIECE & ID_EQUIPEMENT are relationship column. i want to set the value of each one using values of existing row in PIECES but it didn't work

**Reply by melohagan** [reply]
You need to use `_id` for id columns. E.g. `... Block.PIECES.PIECE._id }}`

**Comment by rafikApp** [comment]
for the other columns it works well, but the relationship type columns it doesn't work
![1](https://user-images.githubusercontent.com/106697208/174074888-e06c6933-5efa-435b-9ed6-4c46a007bfc9.PNG)


![2](https://user-images.githubusercontent.com/106697208/174074896-fbc891c2-f5ca-4f41-9263-b7b2c8713314.PNG)

**Reply by melohagan** [reply]
Would you mind sharing your app export to support@budibase.com and then I can take a look

**Comment by rafikApp** [comment]
Thank you so much  it works now

**Comment by rafikApp** [comment]
Can you help me on this ?  #6355

**Reply by melohagan** [reply]
All in due course!
